olp-cpp-sdk-core/src/http/curl/NetworkCurl.cpp

```diff
@@ -919,27 +927,35 @@ size_t NetworkCurl::HeaderFunction(char* ptr, size_t size, size_t nitems,
   if (!that || !that->IsStarted() || handle->cancelled) {
     return len;
   }
+
+  if (!handle->header_callback) {
+    return len;
+  }
+
+  // Drop trailing '\r' and '\n'
   size_t count = len;
   while ((count > 1u) &&
          ((ptr[count - 1u] == '\n') || (ptr[count - 1u] == '\r')))
     count--;
   if (count == 0u) {
     return len;
   }
-  std::string str(ptr, count);
+
+  std::string_view str(ptr, count);
```

Comment:
This project allows only C++11. So, std::string_view could not be used.

Reply:
Got it.

Apart from that item, are you okay with the changes?